### PR TITLE
adds kube publishing rules for origin-4.5-kubernetes-1.18.2

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: publishing-kube-rules.yaml [hunk 1]
```diff
@@ -7,6 +7,145 @@ recursive-delete-patterns:
 - BUILD.bazel
 - "*/BUILD.bazel"
 rules:
+# origin-4.5-kubernetes-1.18.2
```
Comment (marun) [inline publishing-kube-rules.yaml:10]: Is it desirable to add new rules vs updating the rules for 1.18.0-rc.1?

Comment (sttts) [reply publishing-kube-rules.yaml:10]: We remove the old eventually.

Comment (sttts) [reply publishing-kube-rules.yaml:10]: ... when origin rebase merged.

**Pull request merged**